randovania/data/json_data/prime3.txt

```diff
@@ -1092,26 +1163,41 @@ Hillside Vista
 Asset id: 13902157786380317936
 > Dock to Grand Court; Heals? False
   * Normal Door to Grand Court/Dock to Hillside Vista
-  > Dock to Reliquary I
-      Trivial
   > Pickup (Missile Expansion)
       Trivial
+  > Reliquary Ledge
+      Trivial
 
 > Dock to Reliquary I; Heals? False
   * Normal Door to Reliquary I/Dock to Hillside Vista
-  > Dock to Grand Court
-      Grapple Swing or Use Screw Attack (Space Jump)
+  > Reliquary Ledge
+      After Hillside Vista Statue
 
 > Pickup (Missile Expansion); Heals? False
   * Pickup 28; Major Location? False
+  > Reliquary Ledge
+      Trivial
+
+> Reliquary Ledge; Heals? False
+  > Dock to Grand Court
+      Grapple Swing or Use Screw Attack (Space Jump)
   > Dock to Reliquary I
+      After Hillside Vista Statue
+  > Event - Statue
+      Grapple Lasso
```

Comment:
This helper node doesn't seem necessary. Could just have Grand Court connect to Reliquary I with Lasso as before, and have the Missile connect back to Grand Court, and Reliquary I to Grand Court after statue and with Swing or Screw Attack.

Reply:
Done - I also linked Grand Court to Event - Statue with a grapple lasso requirement, then Statue to Grand Court as trivial.